app/controllers/api/v1/posters_controller.rb

```diff
@@ -1,5 +1,60 @@
 class Api::V1::PostersController < ApplicationController
   def index
-    render json: Poster.all
+    posters = Poster.all
+
+    render json: posters.map { |poster|
```

Comment:
This is well done! Let's move this to a serializer later in the project :)

Reply:
That's a great idea! I will need your help with the serializers!